> Can you drop the exact log (at least last few lines) of the expected and actual result? I couldn't find the device in the office but I'd have assumed that the reported issue would appear a bit before that

Sure, of course. Assuming in `RIOT/dist/tools/esptools` there are neither `venv` nor `venv_docker`, I start a build as before. Compilation works fine, after the creation of an Espressif virtualenv. Flashing on the host breaks, though:

```
...
"make" -C /data/riotbuild/riotbase/sys
"make" -C /data/riotbuild/riotbase/sys/auto_init
"make" -C /data/riotbuild/riotbase/sys/div
"make" -C /data/riotbuild/riotbase/sys/frac
"make" -C /data/riotbuild/riotbase/sys/libc
"make" -C /data/riotbuild/riotbase/sys/luid
"make" -C /data/riotbuild/riotbase/sys/newlib_syscalls_default
"make" -C /data/riotbuild/riotbase/sys/pm_layered
"make" -C /data/riotbuild/riotbase/sys/preprocessor
"make" -C /data/riotbuild/riotbase/sys/random
"make" -C /data/riotbuild/riotbase/sys/stdio
"make" -C /data/riotbuild/riotbase/sys/tsrb
"make" -C /data/riotbuild/riotbase/sys/usb_board_reset
"make" -C /data/riotbuild/riotbase/sys/ztimer
esptool v5.0.0
Creating ESP32S3 image...
Merged 3 ELF sections.
Successfully created ESP32S3 image.
# append size of /data/riotbuild/riotbase/examples/basic/blinky/bin/seeedstudio-xiao-esp32s3/blinky.elf.bin
Parsing CSV input...
   text    data     bss     dec     hex filename
  67118   13740  268629  349487   5552f /data/riotbuild/riotbase/examples/basic/blinky/bin/seeedstudio-xiao-esp32s3/blinky.elf
stty -F /dev/ttyACM0 raw ispeed 1200 ospeed 1200 cs8 -cstopb ignpar eol 255 eof 255
sleep 1
/home/riot/RIOT/dist/tools/esptools/venv/bin/esptool --chip esp32s3 --port /dev/ttyACM0 --baud 460800 --before default-reset write-flash -z --flash-mode dio --flash-freq 80m --flash-size detect 0x0000 /home/riot/RIOT/examples/basic/blinky/bin/seeedstudio-xiao-esp32s3/esp_bootloader/bootloader.bin 0x8000 /home/riot/RIOT/examples/basic/blinky/bin/seeedstudio-xiao-esp32s3/partitions.bin 0x10000 /home/riot/RIOT/examples/basic/blinky/bin/seeedstudio-xiao-esp32s3/blinky.elf.bin
make: /home/riot/RIOT/dist/tools/esptools/venv/bin/esptool: No such file or directory
make: *** [/home/riot/RIOT/examples/basic/blinky/../../../Makefile.include:842: flash] Error 127
```

Looking at `RIOT/dist/tools/esptools`, there is only `venv` and no trace of `venv_docker`. The tools are quite obviously those created for the container, for instance  `venv/bin/esptool` refers to `/data/riotbuild/riotbase/dist/tools/esptools/venv/bin/python3`.

--- a/makefiles/docker.inc.mk
+++ b/makefiles/docker.inc.mk
@@ -36,7 +36,8 @@ export DOCKER_MAKECMDGOALS := $(filter $(DOCKER_MAKECMDGOALS_POSSIBLE),$(MAKECMD
 
 # Docker creates the files .dockerinit and .dockerenv in the root directory of
 # the container, we check for the files to determine if we are inside a container.
-ifneq (,$(wildcard /.dockerinit /.dockerenv))
+# Podman uses /run/.containerenv instead.
+ifneq (,$(wildcard /.dockerinit /.dockerenv /run/.containerenv))
   export INSIDE_DOCKER := 1
 else
   export INSIDE_DOCKER := 0

